File: Playwright/.net/TestProject/Model/TodoPage.cs
namespace TestProject.Model
{
    public class TodoPage : PageBase
    {
        public TodoPage(IPage page)
            : base(page, "https://example.cypress.io/todo")
        {
        }

        public ILocator List => Page.Locator("css=.todo-list li");
    }
}

File: Playwright/node.js/model/TodoPage.ts
import { Page } from "playwright-core";
import { PageBase } from "./PageBase";

export class TodoPage extends PageBase {
  constructor(page: Page) {
    super(page, 'https://example.cypress.io/todo');
  };

  get list() {
    return this.page.locator('css=.todo-list li');
  }
};